jetty-server/src/test/java/org/eclipse/jetty/server/ProxyCustomizerTest.java

```diff
@@ -0,0 +1,148 @@
+package org.eclipse.jetty.server;
```

Comment:
missing license header

Reply:
Indeed. I should pay more attention to that.